package/yast2-alternatives.changes

```diff
@@ -1,4 +1,10 @@
 -------------------------------------------------------------------
+Fri Jul 22 11:21:46 UTC 2016 - joaquinmnsm@gmail.com
+
+- Added dekstop file.
```

Comment:
There is a typo in `dekstop`. Other than that, messages in the changelog file should be oriented to final users. "added desktop file" is fine as a commit message (targeted to other developers). But users would appreciate more something like "Added entry in YaST Control Center and system menu". Just an example.